core/src/main/java/org/mule/runtime/core/internal/streaming/ManagedCursorProvider.java

```diff
@@ -19,14 +26,23 @@
  */
 public abstract class ManagedCursorProvider<T extends Cursor> implements CursorProvider<T> {
 
+  private static final Logger LOGGER = getLogger(ManagedCursorProvider.class);
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused "LOGGER" private field. ([squid:S1068](https://sonarqube.kbuild.msap.io/coding_rules#rule_key=squid%3AS1068))

[See it in SonarQube](https://sonarqube.kbuild.msap.io/project/issues?id=mule-runtime-mule&pullRequest=7643&issues=AWirZZm9otIXnmsUDp_R&open=AWirZZm9otIXnmsUDp_R)

[ISSUE_KEY]: AWirZZm9otIXnmsUDp_R